addons/account/wizard/account_resequence.py

```diff
@@ -100,7 +100,7 @@ def _get_move_key(move_id):
             for move in record.move_ids._origin:  # Sort the moves by period depending on the sequence number reset
                 moves_by_period[_get_move_key(move)] += move
 
-            format, format_values = self.env['account.move']._get_sequence_format_param(record.first_name)
+            seq_format, format_values = record.move_ids._origin[0]._get_sequence_format_param(record.first_name)
```

Comment:
Not sure why you need the `_origin`?

Reply:
because of this https://github.com/odoo/odoo/blob/14.0/addons/account/wizard/account_resequence.py#L42

Reply:
I still do not understand

Reply:
I assumed it would be necessary seeing it just above (https://github.com/odoo/odoo/blob/14.0/addons/account/wizard/account_resequence.py#L100). But it's not. Removed it.